src/client/index.js

```diff
@@ -1,8 +1,5 @@
+import App from "./components/App";
+import React from "react";
 import { render } from "react-dom";
-import React from 'react';
-import App from './components/App';
 
-render(
-    <App />,
-    document.getElementById('root')
-);
+render(<App />, document.getElementById("root"));
```

Comment:
just curious since i've seen this a couple times, are you running a formatter plugin or making these one line manually?

Reply:
It's automatic!